### PR TITLE
Document some undocumented template commands.

### PR DESCRIPTION
**Pull request by ahelsley**
Document `cstr`, `unixtime{32,64}`, and the more unusual `uint{8,16,32,64}_bits` template commands.

### CODE REVIEW

File: templates/Reference.md [hunk 1]
```diff
@@ -94,6 +101,7 @@ Various commands are provided for reading and interpreting multiple bytes.
 | ascii *len* *label* | Reads *len* bytes as ASCII | `ascii 32 "Name"` |
 | utf16 *len* *label* | Reads *len* bytes as UTF16 (via current endian) | `utf16 12 "Name"` |
 | str *len* *encoding* *label* | Reads *len* bytes using the specified *encoding* identifier [v2.11+] | `str 8 "utf8" "Name"` |
+| cstr *encoding* *label* | Reads a sequence of zero-termited bytes using the specified *encoding* identifier | `cstr "utf8" "Name"` |
```
Comment (kainjow) [inline templates/Reference.md:104]:
```suggestion
| cstr *encoding* *label* | Reads a sequence of zero-terminated bytes using the specified *encoding* identifier | `cstr "utf8" "Name"` |
```

File: templates/Reference.md [hunk 2]
```diff
@@ -16,22 +16,29 @@ As of v2.10+ the label argument is optional. When the label is not passed, no en
...
+| uint64      | Reads an unsigned 64-bit integer |
+| int64       | Reads a signed 64-bit integer |
+| uint32      | Reads an unsigned 32-bit integer |
+| int32       | Reads a signed 32-bit integer |
+| uint24      | Reads an unsigned 24-bit integer |
+| uint16      | Reads an unsigned 16-bit integer |
+| int16       | Reads a signed 16-bit integer |
+| uint8       | Reads an unsigned 8-bit integer |
+| int8        | Reads a signed 8-bit integer |
+| uint8_bits  | Read some number of `bits` out of an unsigned 8-bit integer |
```
Comment (kainjow) [inline templates/Reference.md:30]: I think we should have the uintx_bits functions as their own section since this doesn't document how to use them. you can leave them off of this PR if you don't want to expand on that for now.

File: templates/Reference.md [hunk 3]
```diff
@@ -16,22 +16,29 @@ As of v2.10+ the label argument is optional. When the label is not passed, no en
...
+| uint16_bits | Read some number of `bits` out of an unsigned 16-bit integer |
+| uint32_bits | Read some number of `bits` out of an unsigned 32-bit integer |
+| uint64_bits | Read some number of `bits` out of an unsigned 64-bit integer |
+| float       | Reads a 32-bit floating point |
+| double      | Reads a 64-bit floating point |
+| uuid        | Reads 16-byte UUID |
+| macdate     | Reads classic Mac OS 4-byte date (seconds since January 1, 1904) |
+| fatdate     | Reads FAT, or DOS, 2-byte date (v2.13+) |
+| fattime     | Reads FAT, or DOS, 2-byte time (v2.13+) |
+| unixtime32  | Reads a UNIX time in (seconds since 00:00:00 UTC on 1 January 1970) |
```
Comment (kainjow) [inline templates/Reference.md:40]: I think we can just shorten this to not show the timestamp. also let's use this format for the date since I don't think the other is commonly used for English(?).
```suggestion
| unixtime32  | Reads a UNIX time in (seconds since January 1, 1970) |
```

File: templates/Reference.md (hunk 2 above)
Comment (ahelsley) [reply templates/Reference.md]: The goal was to get them in since some documentation seemed better than none, but I'm ok with a separate PR for the `_bits` variants.

File: templates/Reference.md [hunk 4]
```diff
@@ -94,6 +97,7 @@ Various commands are provided for reading and interpreting multiple bytes.
 | ascii *len* *label* | Reads *len* bytes as ASCII | `ascii 32 "Name"` |
 | utf16 *len* *label* | Reads *len* bytes as UTF16 (via current endian) | `utf16 12 "Name"` |
 | str *len* *encoding* *label* | Reads *len* bytes using the specified *encoding* identifier [v2.11+] | `str 8 "utf8" "Name"` |
+| cstr *encoding* *label* | Reads a sequence of zero-terminated bytes using the specified *encoding* identifier | `cstr "utf8" "Name"` |
```
Comment (kd35a) [inline templates/Reference.md:100]: I was just going to create a PR adding the same documentation when I found this one. The only difference to my changes is the wording here:
```suggestion
| cstr *encoding* *label* | Reads a sequence of null-terminated bytes using the specified *encoding* identifier | `cstr "utf8" "Name"` |
```

To my knowledge *null-terminated* is the most commonly used terminology for this way of encoding strings. See for example:
* https://en.wikipedia.org/wiki/C_string_handling
* https://en.wikipedia.org/wiki/Null-terminated_string

**Comment by kainjow** [comment]
Thanks!
